### PR TITLE
MHV launch

### PR DESCRIPTION
**Pull request by U-DON**
Resolves department-of-veterans-affairs/vets.gov-team#5094.

Adds back the sign-in button and FAQ content for MHV.

### CODE REVIEW

**Review by bshyong — APPROVED** [review]
LGTM

**Comment by bshyong** [comment]
@U-DON I'll leave this to you to merge whenever you guys are ready to have this in production

**Review by saneshark — APPROVED** [review]
LGTM, lets wait until testing is complete to merge though.

**Pull request merged**
